examples/set-default-labels-nested-package/app2_mysql/README.md

```diff
@@ -0,0 +1,21 @@
+# app2
```

Comment:
Why app1 doesn't have a README but app2 has?

Reply:
Should not need README in both packages. Cleaned up the app2/README.md